Escape object/data names in Yul serializer (#16517)
ethereum/solidity: Changelog.md

* Yul EVM Code Transform: Improve stack shuffler performance by fixing a BFS deduplication issue.

Bugfixes:

> * Yul: Fix incorrect serialization of Yul object names containing special characters (e.g. quotes) that were not escaped, producing output that could not be reparsed.

Reviewer: I would make it more concrete:

```suggestion
* Yul: Fix incorrect serialization of Yul object names containing double quotes and escape sequences, producing output that could not be parsed as valid Yul.
```
